test/unit/thread_event.c

```diff
@@ -9,6 +9,7 @@ TEST_BEGIN(test_next_event_fast_roll_back) {
 	    THREAD_ALLOCATED_NEXT_EVENT_FAST_MAX);
 	thread_allocated_next_event_fast_set(tsd,
 	    THREAD_ALLOCATED_NEXT_EVENT_FAST_MAX);
+	thread_event_recompute_fast_threshold(tsd);
```

Comment:
Should we avoid the `thread_allocated_next_event_fast_set()` call on the previous line?

Reply:
Good catch. Will update.